src/when.test.js

```diff
@@ -109,6 +109,29 @@ describe('When', () => {
       expect(caughtErr.message).toMatch(/Failed verifyAllWhenMocksCalled: 2 not called/)
     })
 
+    it('should print a full stacktrace if verification check fails', () => {
+      const fn1 = jest.fn()
+
+      function extractedWhenConfiguration () {
+        when(fn1).expectCalledWith(expect.anything()).mockReturnValueOnce('z')
+      }
+
+      extractedWhenConfiguration()
+      extractedWhenConfiguration()
+
+      fn1(1)
+
+      let caughtErr
+
+      try {
+        verifyAllWhenMocksCalled()
+      } catch (e) {
+        caughtErr = e
+      }
+
+      expect(caughtErr.message).toContain('at Object.extractedWhenConfiguration')
```

Comment:
Why does it say "Object." first? Not a big deal... I'll still approve it. Just curious.

Reply:
This was the line we found in the stacktace 🤷 my guess is that this is because the referenced function is defined inside the "it object"